examples/jokes/app/routes/jokes.tsx

```diff
@@ -21,7 +21,10 @@ export let loader: LoaderFunction = async ({ request }) => {
 
   let data: LoaderData = {
     jokeListItems,
-    user,
+    user: user && {
+      id: user.id,
+      username: user.username,
+    },
```

Comment:
The `getUser` function could possibly use [Prisma's `select`](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#select) property:


```js
async function getUser() {
  return db.user.findUnique({
    where: { /* ... */ },
    select: {
      id: true,
      username: true
    }
  })
}
```

Reply:
Yes that will also fix it but will limit the usage of getUser.

Reply:
true! Although since it's only 1 property of the user not being selected this could be a parameter in the function, but this might make the function too complex for a simple example 😅.